R/run_stress_test.R

```diff
@@ -371,9 +374,20 @@ st_process <- function(data, asset_type) {
   sector_exposures <- data$sector_exposures %>%
     process_sector_exposures(asset_type = asset_type)
 
+  capacity_factors_power <- data$capacity_factors %>%
+    process_capacity_factors_power(
+      scenarios_filter = scenarios_filter,
+      scenario_geography_filter = scenario_geography_filter_lookup,
+      technologies = technologies_lookup,
+      start_year = start_year,
+      end_year = end_year_lookup
+    )
```

Comment:
I see we go from `capacity_factor` to `capacity_factor_power`. Is this correct? I assume we read data from every sector, then process only data in the power sector. If that's the case, I would prefer to expose the step that picks the power sector specifically. Does it make sense?

```r
data$capacity_factors %>%
  pick_power() %>%
  process_capacity_factor_power()
```

Reply:
Apparently capacity_factors (currently) only holds capacity_factors for the power sector ATM. This might be extended but I have no background knowledge on how likely and how soon that would happen, maybe @jacobvjk  knows more.
While the status quo is as it is I would suggest renaming read_capacity_factors to read_capacity_factors_power. If doing so please make sure it does not interfere with the webtool though/adjust webtool accordingly.

Reply:
I defer it to https://github.com/2DegreesInvesting/r2dii.climate.stress.test/issues/307.